.github/workflows/build-switch.yml

```diff
@@ -0,0 +1,26 @@
+name: Build for Nintendo Switch using CMake provided by devkitPro
+on:
+  pull_request:
+  push:
+  release:
+    types: published
+jobs:
+  build-nintendo-switch:
+    runs-on: ubuntu-latest
+    container: devkitpro/devkita64:latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: "Build files"
+        run: |
+          /opt/devkitpro/portlibs/switch/bin/aarch64-none-elf-cmake -S. -Bbuild .. -DLIBRW_PLATFORM=GL3 -DLIBRW_GL3_GFXLIB=GLFW -DLIBRW_INSTALL=True
```

Comment:
Does it work with the SDL2 backend?

Reply:
Oh, it's missing indeed. It should work though, I'll add it in a while.